wp/lib/bap_wp/src/constraint.ml

```diff
@@ -305,16 +350,16 @@ let get_refuted_goals ?filter_out:(filter_out = []) (constr : t)
         worker c2 current_path current_registers olds news
       else
         failwith (Format.sprintf "get_refuted_goals: Unable to resolve branch \
-                                    condition %s at %s"
-                                    (Z3.Expr.to_string cond_res)
-                                    (jmp |> Term.tid |> Tid.to_string))
+                                  condition %s at %s"
+                    (Z3.Expr.to_string cond_res)
+                    (jmp |> Term.tid |> Tid.to_string))
```

Comment:
ocp-ident keeps changing this string. Is this a problem @fortunac ?

Reply:
Nope! Should be fine.